lib/views/general/_default_footer.text.erb

```diff
@@ -3,8 +3,8 @@
 WhatDoTheyKnow is a project of mySociety run by a small team of staff and
 dedicated volunteers.
 
-You can help to make information accessible to all by donating today.
-Your donations, however small, really help:
+mySociety is a charity that believes that information should be accessible to everyone.
+If this is important to you, please consider donating to help us do more:
```

Comment:
I realise this is a little long, but it didn't seem like it would flow as well if there was another linebreak (e.g. before 'accessible').